internal/servers/controller/handlers/groups/group_service.go

```diff
@@ -247,24 +248,26 @@ func (s Service) RemoveGroupMembers(ctx context.Context, req *pbs.RemoveGroupMem
 }
 
 func (s Service) getFromRepo(ctx context.Context, id string) (*pb.Group, error) {
+	const op = "groups.(Service).getFromRepo"
 	repo, err := s.repoFn()
 	if err != nil {
-		return nil, err
+		return nil, errors.Wrap(err, op)
 	}
 	g, m, err := repo.LookupGroup(ctx, id)
 	if err != nil {
 		if errors.IsNotFoundError(err) {
-			return nil, handlers.NotFoundErrorf("Group %q doesn't exist.", id)
+			return nil, nil
 		}
-		return nil, fmt.Errorf("unable to get group: %w", err)
+		return nil, errors.Wrap(err, op)
 	}
 	if g == nil {
-		return nil, handlers.NotFoundErrorf("Group %q doesn't exist.", id)
+		return nil, errors.New(errors.InvalidParameter, op, fmt.Sprintf("group %q not found", id))
```

Comment:
In this PR https://github.com/hashicorp/boundary/pull/1077 we upper cased error messages that would end up as the first message to the user